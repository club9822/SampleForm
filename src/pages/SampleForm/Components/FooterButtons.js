import React, {Component, useEffect, useState, useReducer} from 'react';
import {
  StyleSheet,
  Text,
  View,
  TouchableWithoutFeedback,TouchableOpacity
} from 'react-native';
import {width, height, isIphoneX} from '../../../utils/windowDimensions';

/**
 *
 * @param props
 * @return {*}
 * @constructor
 */

// @flow
type Props = {
};
function FooterButtons(props:Props){
  return (
    <View style={{width:'100%',minHeight:'10%',alignItems:'center',justifyContent:'center'}}>
      <TouchableOpacity

        onPress={() => {
          alert('00');
        }}>

        <Text style={{fontFamily: 'IRANSansMobile(FaNum)'}}>ثبت نام</Text>
      </TouchableOpacity>
    </View>
  );
}

export default FooterButtons;
