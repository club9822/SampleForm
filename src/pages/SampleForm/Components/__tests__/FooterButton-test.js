import React from 'react';
import FooterButtons from '../FooterButtons';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<FooterButtons />).toJSON();
  expect(tree).toMatchSnapshot();
});
