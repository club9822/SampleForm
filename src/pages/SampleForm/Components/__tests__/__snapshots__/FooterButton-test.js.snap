// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "justifyContent": "center",
      "minHeight": "20%",
    }
  }
>
  <Text
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "fontFamily": "IRANSansMobile(FaNum)",
      }
    }
  >
    ثبت نام
  </Text>
</View>
`;
